test/TritonGPU/pipeline-hopper-remove-wait.mlir

```diff
@@ -1,5 +1,6 @@
-// RUN: triton-opt %s -split-input-file -tritongpu-rewrite-tensor-pointer -canonicalize -tritongpu-pipeline=compute-capability=90 -canonicalize | FileCheck %s
+// RUN: triton-opt %s --verify-each -tritongpu-rewrite-tensor-pointer -canonicalize -tritongpu-pipeline=compute-capability=90 -canonicalize | FileCheck %s
 
+// XXX This one appears to have a real bug?
```

Comment:
So the IR is invalid? That's surprising as this is something added recently. Disabling verifier means we potentially miss other problems in the IR generated that seems a bit risky.
What is the problem?

Reply:
Same as above; I've reverted changes in this file.  Didn't see my own "XXX" in there.  Maybe I need a presubmit...